### DISCUSSION TITLE
Wrong dj username received by Liquidsoap using FFmpeg

### DISCUSSION THREAD

**Discussion by mimmosoft** [AzuraCast/AzuraCast#4084] [Q&A]
Hi everyone.
I'm facing this issue: i'm trying to start a live stream using ffmpeg but i can't get it working.
I send this command:
`ffmpeg -re -i /www/wwwroot/recordings/test.mp3 -vn -c:a libmp3lame -b:a 128k -legacy_icecast 1 -content_type audio/mpeg -ice_name "Test" -f mp3 icecast://mydjuser:mydjpass@myserver:8006/radio.mp3`
and i receive this response: Error writing trailer of icecast://mydjuser:mydjpass@myserver:8006/radio.mp3: Connection reset by peer

So, i gave a look to the Liquidsoap log for that station and i noticed that the dj user received is "source" and not "mydjuser":
`2021/04/27 11:42:17 [lang:3] Authenticating DJ: source
2021/04/27 11:42:17 [lang:3] AzuraCast DJ Auth Response: false`

I can't figure out where i'm wrong. Can someone help me?
Thanks

**Comment by r0nd0n** [comment]
I would be interested in this as well. I have been trying to do this with icecast instead of the shoutcast instance, as I use ffmpeg with standalone icecast2 on another server with no problems, but can't get it to work with Azuracast. I get either a `av_interleaved_write_frame(): Broken pipe` error, or `Server returned 404 Not Found` error from the terminal, with no useful info in the icecast access or error logs on Azuracast.
